golem-rib/src/compiler/ir.rs

```diff
@@ -195,6 +169,32 @@ mod protobuf {
     use golem_wasm_ast::analysis::{AnalysedType, TypeStr};
     use golem_wasm_rpc::ValueAndType;
 
+    impl From<golem_api_grpc::proto::golem::rib::WorkerNamePresence> for WorkerNamePresence {
+        fn from(value: golem_api_grpc::proto::golem::rib::WorkerNamePresence) -> Self {
+            match value {
+                golem_api_grpc::proto::golem::rib::WorkerNamePresence::Present => {
+                    WorkerNamePresence::Present
+                }
+                golem_api_grpc::proto::golem::rib::WorkerNamePresence::Absent => {
+                    WorkerNamePresence::Absent
+                }
+            }
+        }
+    }
+
+    impl From<WorkerNamePresence> for golem_api_grpc::proto::golem::rib::WorkerNamePresence {
+        fn from(value: WorkerNamePresence) -> Self {
+            match value {
+                WorkerNamePresence::Present => {
+                    golem_api_grpc::proto::golem::rib::WorkerNamePresence::Present
+                }
+                WorkerNamePresence::Absent => {
+                    golem_api_grpc::proto::golem::rib::WorkerNamePresence::Absent
+                }
+            }
+        }
+    }
```

Comment:
I think you can remove this structure both from proto and code. I forgot to may be remove it. If you could try, that will be good since you are anyways touching these parts